### DISCUSSION TITLE
Repeater associate existing records

### DISCUSSION THREAD

**Discussion by michele-grifa** [filamentphp/filament#10656] [Help]
Is it possible to use a repeater to associate existing records in a hasMany relationship, instead of creating new ones?

**Comment by leandrocfe** [comment]
Why not a relation manager?
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#associating-and-dissociating-records

**Reply by michele-grifa** [reply]
For UI purposes, I have a form with tabs and in one of them I want to insert this relationship association.
